### DISCUSSION TITLE
ImageMagic Generation for Multiple Languages

### DISCUSSION THREAD

**Discussion by DavidKrassnig** [george-gca/multi-language-al-folio#7] [General]
Hi @george-gca,

There's something I noticed which I wanted to ask you about: when you generate a site with multiple languages, ImageMagic generates scaled down versions of each image for each language. This causes a massive slowdown for n>2 languages (not to mention a waste of space).

Regular, non-downscaled images are correctly read directly from the `assets/img/` folder without localisation; but the ImageMagick plugin seems to not have gotten the memo. I've taken a look, but I can't see an obvious way on how to fix this. Any ideas?

**Edit:** I mean, one could always just disable the plugin, but apparently some people get issues when this is done? (the `_config.yml` refers to issue [#537 from alshedivat/al-folio](https://github.com/alshedivat/al-folio/issues/537))

**Comment by george-gca** [comment]
I have never noticed this. Good point, I will definitely take a look, but currently no idea.

**Comment by george-gca** [comment]
Fixed this with [this commit](https://github.com/george-gca/multi-language-al-folio/commit/f2520a1be72c01355309033b01b5b51e32299bbc).

**Reply by george-gca** [reply]
Even though it still displays in terminal that it is converting images from `_site/pt-br/`, these files are no longer in `_site/pt-br/assets/`. Don't know why it is still printing.
